.vscode/tasks.json

```diff
@@ -0,0 +1,13 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
```

Comment:
JSON doesn't support comments, does vscode just ignore them?

Reply:
yes

Reply:
ironically this is a generated file from VSCode :/ I'll remove the comment